Task3.java

```diff
@@ -0,0 +1,19 @@
+public class Task3 {
+
+    // Створити метод stringReverse. Метод приймає як параметр - рядок.
+    // Необхідно її розгорнути та повернути змінений варіант.
+    // Наприклад:
+    // Hello -> olleH
+
+
+    public static void main(String[] args) {
+
+        System.out.println("The original string is: " + "Apple" +
+                "\nThe reversed String is: " + stringReverse("Apple"));
+    }
+
+    public static String stringReverse(String str) {
+

```

Comment:
Empty lines should be removed as it is unnecessary to have them here. This is applicable to all the tasks, so please remove them.